internal/environment/lab/network/dhcp/dhcp.go

```diff
@@ -30,6 +31,7 @@ func New(format func(n int) string) (*Server, error) {
 
 	subnet := format(0)
 	dns := format(dns.PreferedIP)
+	// TODO: Test how resetting could affect docker IPs
```

Comment:
It could create routing issues if two machines has the same ip address as the packets going to the IP will be send randomly to one of the machines

Reply:
I will put thoroughly testing this in the backlog for later